fem/geom_ext.cpp

```diff
@@ -245,15 +152,15 @@ Geometry* Geometry::Get(const FiniteElementSpace& fes,
       geom->invJ.allocate(dims, dims, numQuad, elements);
```

Comment:
This looks like a lot of memory! I have doubt on the utility to store the `invJ`, storing only `detJ` makes sense if we don't store `J` and `invJ`. The risk here is that you run out of memory before being anywhere close to the amount of elements you need to be efficient on a GPU.